staging/src/k8s.io/kubectl/pkg/cmd/certificates/certificates.go

```diff
@@ -33,15 +33,16 @@ import (
 	"k8s.io/cli-runtime/pkg/genericclioptions"
 	"k8s.io/cli-runtime/pkg/printers"
 	"k8s.io/cli-runtime/pkg/resource"
-	clientset "k8s.io/client-go/kubernetes"
+	v1 "k8s.io/client-go/kubernetes/typed/certificates/v1"
```

Comment:
Nit: use `certificatesv1client` alias, same below, `certificatesv1beta1client` to make it more readable in the code.